gatsby-config.js

```diff
@@ -72,5 +72,8 @@ module.exports = {
         exclude: ['/imprint', '/data-privacy', '/blog/**'],
       },
     },
+    {
+      resolve: `gatsby-plugin-netlify`,
```

Comment:
wow how much code goes into that plugin?
https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-netlify/src

I'm a fan of cherry picking the small bits we need. Can't be that much in this case. Would this be possible instead of pulling in a plugin that does ton of things we don't realize (which could bite us in the future).

It's okay for this PR though, I'm just wondering if you would be willing to share my opinion about plugins like this.

Reply:
Actually I prefer the plugin over copying code into our repo:
- As it's an official plugin, it's pretty save that it will always be up-to-date with Gatsby 
- It creates a quite sophisticated `_headers` file which covers all files and not just the fonts. I wouldn't want to implement and maintain all of the logic on my own